### PR TITLE
SNAP Standard Deduction

### LINKED ISSUE #73: Standard deduction SNAP

### PR DESCRIPTION
**Pull request by nmrodelo**
fixes #73, add SNAP Standard Deduction by Family Size

### CODE REVIEW

File: openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+      2021-01-01: 167
+    3:
+      2021-01-01: 167
+    4:
+      2021-01-01: 181
+    5:
+      2021-01-01: 212
+    6:
+      2021-01-01: 243
+  AK_urban:
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml:18]: Could you add a comment that values are identical across the Alaska subgeographies?

Comment (MaxGhenis) [reply openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml:18]: Per our meeting, combine to AK

File: openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml [hunk 2]
```diff
@@ -0,0 +1,104 @@
+description: Maximum Monthly Shelter Deduction by State and Household Size
+
+#NOTE: household sizes above 6 use the rate for a 6 member household in all states.   
+standard_deduction:
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml:4]: delete line 4 and unindent

File: openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml [hunk 3]
```diff
@@ -0,0 +1,104 @@
+description: Maximum Monthly Shelter Deduction by State and Household Size
```
Comment (MaxGhenis) [inline openfisca_us/parameters/benefit/SNAP/standard_deduction.yaml:1]: make closer to max allotment description

**Review by MaxGhenis — COMMENTED** [review]
rename to `SNAP/deductions/standard.yaml`

**Review by nikhilwoodruff — APPROVED** [review]

**Pull request merged**